### DISCUSSION TITLE
How to have a relation based on the parent relation?

### DISCUSSION THREAD

**Discussion by geosigno** [strapi/strapi#7492] [Legacy Topics]
I have 3 collections:
1. lesson
2. unity
3. level

with these 3 relations:
1. one level to many lessons 
2. one level to many units
3. one unity to many lessons

Is it possible to have the list of units available based on the level of that course while creating a lesson? Right now, all units are available despite the level of that lesson.

My issue is that 2 different units will have the same name ("Unity 1" can belong to "Level 1", and another "Unity 1" can belong to another "Level 2"). So it is not understandable via the Unity select list. 
![Capture d’écran 2020-08-16 à 23 39 20](https://user-images.githubusercontent.com/6083654/90342507-e2acc880-e019-11ea-9ab3-f425b02a0b87.png)

**Comment by derrickmehaffy** [comment]
Hi @geosigno conditional relations aren't currently a feature within Strapi yet, this is something you would need to build yourself by extending the `strapi-plugin-content-manager` package or you can [submit a feature request](https://github.com/strapi/strapi/issues/new?template=FEATURE_REQUEST.md) on Github/Product Board
